packages/client/components/promptResponse/PromptResponseEditor.tsx

```diff
@@ -68,7 +68,7 @@ const CancelButton = styled(SubmitButton)({
 const StyledEditor = styled('div')`
   .ProseMirror {
     min-height: 40px;
-    line-height: 20px;
+    line-height: 1.25;
```

Comment:
this improves the way header elements are displayed, with line-height hardcoded in pixels there's no room for bigger font sizes